### ISSUE TITLE
Support for static filesystem volume(set) names?

### ISSUE THREAD

**Issue by sysadmin1337** [ClusterHQ/fli#5]
Running `fli create volumesetname volumename` creates uuid paths inside the mounted zpool like `/chq/fc3d6776-d76a-4d90-8ec3-89f348b3536d/ba603e7a-e529-4aa4-b126-7989432c3662`. 

For automation it's not suitable, because you have to manually change the docker volume path to point to a volume in the volumeset(I guess?). Is there going to be support for static volumeset/volume paths? Or am I confused as to how docker volumes should work together with fli?

**Comment by DasAng** [comment]
Yeah I think a best practice guide on how to mount docker volumes with fli could help

**Comment by pcgeek86** [comment]
Hello @sysadmin1337, we have a tool called `fli-docker` that enables you to inject volume paths into a Docker Compose. 

You can also do a simple text replacement operation in your `docker run -v <host>:<container>` command, and inject the Volume mount point. A simple Python or Ruby wrapper would suffice, or even simply setting a Bash environment variable.

Here's an end-to-end Bash script that I wrote to demonstrate the core capabilities of Fli + FlockerHub. This should help push you along. 😄 

https://gist.github.com/pcgeek86/79cbc230e0478eaf9ad5157867bd1f64

FYI, we'll write a blog post about this particular scenario on the ClusterHQ blog, and discuss your request.

I agree with you that we should make it easier to specify a custom mount point, but keep in mind that it must reside within a ZFS Pool.

Please continue to send feedback, as we continue to improve.

Cheers,
Trevor Sullivan
Developer Evangelist, ClusterHQ
https://clusterhq.com

**Comment by wallnerryan** [comment]
@DasAng @sysadmin1337 https://github.com/ClusterHQ/fli-docker is what @pcgeek86 referenced
